### PR TITLE
port over course offerings

### LINKED ISSUE #10: Port over existing course offerings
#### Describe Problem
Existing course offerings
#### Suggest Solution
port em over

### PR DESCRIPTION
**Pull request by prairir**
### What are you trying to accomplish?
port over course offerings and some minor fixes including adding a question to the PR template and moving the style guide

### How are you accomplishing it?
porting over the course offerings has 2 big parts to it. 1 enabling side bars to be foldable, im not sure how this will effect accesibility but it shouldnt. and disabling the side table of contents in the courses section. with it enabled the div cuts off the table making it look weird, disabling it fixes that.

i also moved the style guide to `/guides` because thats where it will live

i also added a question to the PR template if they read and followed the requirements

### Is there anything reviewers should know?
naw

- [x] Is it safe to rollback this change if anything goes wrong?

### CODE REVIEW

**Comment by amanpatel123** [comment]
@prairir any chance you can have a snapshot of how do these changes look?

File: courses/index.md [hunk 1]
```diff
@@ -3,6 +3,72 @@ id: index
...
+# Offered courses
+
+### Requirements Legend
+
+❤️Bachelor of Computer Science (General)<br/>
+💙Bachelor of Computer Science (Honours)<br/>
+💛Bachelor of Computer Science (Honours Applied Computing)<br/>
+💚Bachelor of Science (Honours Computer Science with Software Engineering Specialization)<br/>
+💜Bachelor of Commerce (Honours Business Administration and Computer Science)<br/>
+🖤Bachelor of Mathematics (Honours Mathematics and Computer Science)<br/>
```
Comment (amanpatel123) [inline courses/index.md:18]: I thought we once talked about changing this heart to some other representations, isn't it? Do we do that in new PR?

Comment (prairir) [reply courses/index.md:18]: it looks exactly like the old one but on the new page. I thought it was best to put a quick port up first and then work on the new features.

Comment (amanpatel123) [reply courses/index.md:18]: Ok sounds good

**Review by amanpatel123 — APPROVED** [review]

**Pull request merged**